process-automation/latest/model/connectors/aws.md

```diff
@@ -160,9 +468,9 @@ The input parameters of the Rekognition connector are:
 | Parameter | Type | Description |
 | --------- | ---- | ----------- |
 | file | File | *Required.* A [variable]({% link process-automation/latest/model/processes/index.md %}#process-variables) of type file to send for analysis. |
-| mediaType | String | *Optional.* The media type of the file to be analyzed, for example `application/octect-stream`. |
+| mediaType | String | *Optional.* The media type of the file to be analyzed, for example ``/octect-stream`. |
```

Comment:
Looks like too many ticks around the /octet-stream

Reply:
fixed.